frontend/src/components/maps/leaflet/InfoSlideOut/HeaderActions.tsx

```diff
@@ -79,26 +80,30 @@ const HeaderActions: React.FC<IHeaderActions> = ({
           >
             View details
           </Link>
-          <VerticalBar />
-          <Link
-            onClick={() => {
-              jumpToView();
-              if (onLinkClick) onLinkClick();
-            }}
-            to={{
-              pathname: `/mapview`,
-              search: queryString.stringify({
-                ...queryString.parse(location.search),
-                disabled: false,
-                sidebar: true,
-                loadDraft: false,
-                buildingId: buildingId,
-                parcelId: parcelId,
-              }),
-            }}
-          >
-            Update
-          </Link>
+          {canEdit && (
```

Comment:
So I'm aware that if a user doesn't belong to a properties agency they won't see any of these actions, but if a user does belong and a property is in a project they should be able to view but not update the property.